Add database replica source to `Database/Data` (#28)
osu-tournament-rating/otr-docs: docs/topics/Data.md

# Data

The o!TR team maintains a publicly accessible archive of database replicas to support two key objectives:

1. Facilitating development and contributions to the official [o!TR repositories](https://github.com/osu-tournament-rating/).

> 2. Ensuring interested parties can verify a tournament (which used o!TR to seed or filter registrants) did so in a legitimate manner.

Reviewer: ```suggestion
2. Ensuring that interested parties may verify a tournament using o!TR to seed or filter registrants did so legitimately.
```

## Public Replicas

> Database replicas are updated weekly and stored in a Google Cloud storage bucket. **The public may browse available replicas [here](https://data.otr.stagec.xyz/).**

Reviewer: do you want to specify a weekly time that they will be created and/or made available? doesn't have to be in this doc itself

These public replicas omit the following information due to security and user privacy concerns:

1. Admin notes
2. Audit logs

> 3. Logs

Reviewer: might be worth clarifying the distinction between "audit logs" and "logs"?

also "users" below
